sparse/file.go

```diff
@@ -220,10 +220,5 @@ func GetFiemapRegionExts(file FileIoProcessor, interval Interval) ([]Extent, err
 		lastExtStart = ext.Logical
 	}
 
-	// last ext should have the FIEMAP_EXTENT_LAST set, otherwise we don't get all exts
-	if exts[len(exts)-1].Flags&FIEMAP_EXTENT_LAST == 0 {
```

Comment:
tried logging file ext count out when this failed on Drone. Found that file ext count and region ext count are all equal to 1, so this should not fail no matter where tests run as long as File System implementations of this flag are the same. At this point, I have no conclusion for the weirdness yet.